patches/gcc/10.2.0/0017-x86_64-zephyr-elf-fix-soft-float-for-x86-32-bit.patch

```diff
@@ -0,0 +1,54 @@
+From df5ca42adff42aff861d4af132c87a3b6aaaf21c Mon Sep 17 00:00:00 2001
+From: Daniel Leung <daniel.leung@intel.com>
+Date: Thu, 10 Dec 2020 10:31:19 -0800
+Subject: [PATCH 17/17] x86_64-zephyr-elf: fix soft float for x86 32-bit
+
+This fixes soft-float build for x86 32-bit (-m32 -msoft-float)
+under x86_64-zephyr-elf multilib build. This now actually
+includes the soft float functions.
+
+Signed-off-by: Daniel Leung <daniel.leung@intel.com>
+---
+ libgcc/config.host             | 9 ++++++++-
+ libgcc/config/i386/64/t-softfp | 2 ++
+ 2 files changed, 10 insertions(+), 1 deletion(-)
+ create mode 100644 libgcc/config/i386/64/t-softfp
+
+diff --git a/libgcc/config.host b/libgcc/config.host
+index 54ae74350..f76f23943 100644
+--- a/libgcc/config.host
++++ b/libgcc/config.host
+@@ -704,7 +704,7 @@ i[34567]86-*-elf*)
+ 	tmake_file="$tmake_file i386/t-crtstuff t-crtstuff-pic t-libgcc-pic"
+ 	;;
+ x86_64-zephyr-elf)
+-	tmake_file="$tmake_file i386/t-crtstuff i386/64/t-zephyr64 t-dfprules"
++	tmake_file="$tmake_file i386/t-crtstuff i386/64/t-zephyr66"
```

Comment:
Looks like a typo at the end here `i386/64/t-zephyr66` -> `zephyr64`?

Reply:
Fixed.

Weird that it built without any issue on my system.

Reply:
Same here.  Just noticed as I was about to commit the change :)